docs/language/repl.rst

```diff
@@ -62,6 +62,59 @@ Once an exception has been thrown in an interactive session, the most recent exc
   <class 'ZeroDivisionError'>
 
 
+Initialization Script
+^^^^^^^^^^^^^^^^^^^^^^
+
+Similarly to python's :py:envvar:`PYTHONSTARTUP` environment variable, when
+``HYSTARTUP`` is set, Hy will try to execute the file and import/require its defines
+into the repl namespace. This can be useful to set the repl ``sys.path`` and make
+certain macros and methods available in any Hy repl.
+
+In addition, init scripts can set defaults for repl config values with:
+
+``repl-spy``
+  (bool) print equivalent Python code before executing.
+
+``repl-output-fn``
+  (callable) single argument function for printing REPL output.
+
+Init scripts can do a number of other things like set banner messages or change the
+prompts. The following shows a number of possibilities::
+
+  ;; Wrapping in an `eval-and-compile` ensures global python packages
+  ;; are available in macros defined in this file as well.
+  (eval-and-compile
+    (import sys os)
+    (sys.path.append "~/<path-to-global-libs>"))
+
+  ;; These modules, macros, and methods are now available in any Hy repl
+  (import
+    re
+    json
+    [pathlib [Path]]
+    [hy.contrib.pprint [pp pformat]])
+
+  (require
+    [hy.extra.anaphoric [%]])
+
+  (setv
+    ;; Spy and output-fn will be set automatically for all hy repls
+    repl-spy False
```

Comment:
minor nitpick, wouldn't `True` make more sense as an example?

Reply:
It would! I'll change it next time in front of my work computer

Reply:
done